kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/util/KafkaContainerTest.java

```diff
@@ -0,0 +1,22 @@
+package org.axonframework.extensions.kafka.eventhandling.util;
+
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * A single node of kafka using test container.
```

Comment:
Same argumentation on this guy as I placed on the `KafkaContainerClusterTest` class.